win32/devdoc/file_win32_requirements.md

```diff
@@ -77,17 +78,17 @@ MOCKABLE_FUNCTION(, void, file_destroy, FILE_HANDLE, handle);
 
 **SRS_FILE_WIN32_43_049: [** If `handle` is `NULL`, `file_destroy` shall return. **]**
 
-**SRS_FILE_WIN32_43_011: [** `file_destroy` shall wait for all I/O to complete. **]**
+**SRS_FILE_WIN32_43_011: [** `file_destroy` shall wait for all I/O to complete by calling `WaitForThreadpoolIoCallbacks`. **]**
 
-**SRS_FILE_WIN32_43_012: [** `file_destroy` shall close the cleanup group. **]**
+**SRS_FILE_WIN32_43_012: [** `file_destroy` shall close the cleanup group by calling `CloseThreadpoolCleanupGroup`. **]**
 
-**SRS_FILE_WIN32_43_013: [** `file_destroy` shall destroy the environment. **]**
+**SRS_FILE_WIN32_43_013: [** `file_destroy` shall destroy the environment by calling `DestroyThreadpoolEnvironment`. **]**
 
-**SRS_FILE_WIN32_43_016: [** `file_destroy` shall call `CloseHandle` on `handle` **]**
+**SRS_FILE_WIN32_43_016: [** `file_destroy` shall call `CloseHandle` on `handle->h_file` **]**

```

Comment:
>h_file [](start = 81, length = 6)

Implementation detail (h_file isn't in the spec). Better to just say "on the handle returned by CreateFileA" or something

Reply:
Done

---
In reply to: [456703046](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456703046) [](ancestors = 456703046)